tests/end2end/features/navigate.feature

```diff
@@ -18,6 +18,22 @@ Feature: Using :navigate
         And I run :navigate up with count 2
         Then data/navigate should be loaded
 
+    Scenario: Navigating up in qute://help/
+        When I open qute://help/commands.html
+        And I run :navigate up
+        Then qute://help/ should be loaded
+
+    Scenario: Navigating up by count in qute://help/
+        When I open qute://help/img/cheatsheet-big.png
+        And I run :navigate up with count 2
+        Then qute://help/ should be loaded
+
+    Scenario: Navigating up in qute://help/img/cheatsheet-big.png
```

Comment:
This seems to [fail](https://travis-ci.org/qutebrowser/qutebrowser/jobs/302003894#L676) on the CI, probably because it needs the generated documentation to work? I think it's okay to just try and open `qute://help/img/` or so directly.

Reply:
I'll remove this.